source/PlayerInfo.cpp

```diff
@@ -1203,8 +1207,7 @@ void PlayerInfo::Land(UI *ui)
 	// do not name them all (since this would overflow the screen).
 	else if(ui && !inactiveMissions.empty())
 	{
-		string message = "These active missions or jobs were deactivated due to a missing definition"
-			" - perhaps you recently removed a plugin?\n";
+		string message = "These active missions or jobs were deactivated due to a missing definition - perhaps you recently removed a plugin?\n";
```

Comment:
Some style fixes also appear to be lost here and on the lines below. Please ensure that the fixes from other PRs are not lost.

Reply:
Fixed in follow-up commit.